tv_apps/smart-system/js/child_window_factory.js

```diff
@@ -132,6 +132,24 @@
     return true;
   };
 
+  ChildWindowFactory.prototype.createPreviewWindow = function(evt) {
+    if (this.app.frontWindow &&
+        (this.app.frontWindow.isTransitioning() ||
+          this.app.frontWindow.isActive())) {
```

Comment:
nit: add a comment to indicate we need some error handling here.
`//XXX should we show a hint to tell user we can't preview at this moment?`